packages/jest-core/src/TestScheduler.ts

```diff
@@ -342,9 +343,15 @@ class TestScheduler {
       switch (reporter) {
         case 'default':
           summary = true;
-          verbose
-            ? this.addReporter(new VerboseReporter(this._globalConfig))
-            : this.addReporter(new DefaultReporter(this._globalConfig));
+          if (verbose) {
+            this.addReporter(new VerboseReporter(this._globalConfig));
+          } else {
+            GITHUB_ACTIONS
+              ? this.addReporter(
+                  new GithubActionsLogsReporter(this._globalConfig),
+                )
+              : this.addReporter(new DefaultReporter(this._globalConfig));
+          }
```

Comment:
I don't think this should change

Reply:
Wasn't the objective to add this reporter as the new default if jest is used inside a GITHUB ACTIONS environment? If so, I feel as if this is the cleanest change to make it happen, isn't it? The other option would be to include what the new class does directly inside the default reporter but it seems to me a useless complexity.